layouts/_default/_markup/render-codeblock-math.html

```diff
@@ -0,0 +1,3 @@
+<p>$$
+{{ .Inner | safeHTML }}
+$$</p>
```

Comment:
This code fails if custom delimiters other than `$$` are defined in `config`.`yaml`|`toml`|`json.` I think we should take this into account and improve the code above. What about this:

```
{{/* set default delimiters */}}
{{ $delimiter_left := "$$" }}
{{ $delimiter_right := "$$" }}

{{/* override delimiters if set in config file */}}
{{ with $.Page.Site.Params.katex.options.delimiters }}
  {{ range first 1 ( where . "display" true ) }}
    {{ $delimiter_left = index . "left" }}
    {{ $delimiter_right = index . "right" }}
  {{ end }}
{{end}}

{{/* output of equation */}}
<p>{{ $delimiter_left }}
{{ .Inner | safeHTML }}
{{ $delimiter_right }}</p>
```

Reply:
Good point, added your code snippet